SecurityPkg/Tcg/Tcg2Dxe/Tcg2Dxe.c

```diff
@@ -2497,14 +2501,32 @@ MeasureSecureBootPolicy (
   IN VOID       *Context
   )
 {
-  EFI_STATUS  Status;
-  VOID        *Protocol;
+  EFI_STATUS    Status;
+  VOID          *Protocol;
+  DEVICE_STATE  CurrentDeviceState; // MU_CHANGE - Measure Firmware Debugger Enabled
 
   Status = gBS->LocateProtocol (&gEfiVariableWriteArchProtocolGuid, NULL, (VOID **)&Protocol);
   if (EFI_ERROR (Status)) {
     return;
   }
 
+  // MU_CHANGE [BEGIN] - Measure Firmware Debugger Enabled
+  CurrentDeviceState = GetDeviceState ();
+
+  if ((CurrentDeviceState & DEVICE_STATE_SOURCE_DEBUG_ENABLED) != 0) {
+    Status = MeasureLaunchOfFirmwareDebugger ();
+    if (EFI_ERROR (Status)) {
+      DEBUG ((DEBUG_ERROR, "Failed to measure Firmware Debugger Enabled!\n"));
+      ASSERT_EFI_ERROR (Status);
+
+      PanicReport (__FILE__, __LINE__, "Failed to measure Firmware Debugger Enabled!\n");
+      // TODO cap PCR?

```

Comment:
Handle TODO?

Reply:
I did the thing where I commit amend and push, but didn't actually add the files I wanted to commit :)